### ISSUE TITLE
Release notes 

### ISSUE THREAD

**Issue by Vr93** [ks93/xlogger#13]
Should create release notes once the library has an "official" release with v1.0?

**Comment by ks93** [comment]
Yes, maybe even before - because I think it can take some time 😏
